src/org/omegat/util/Language.java

```diff
@@ -774,26 +774,11 @@ public boolean isSameCountryLanguage(String target) {
         return isSameCountryLanguage(new Language(target));
     }
 
-    /**
-     * Check if locale is Right-To-Left oriented.
-     * @return true if locale is Right-To-Left oriented.
-     */
-    public static boolean localeIsRTL() {
```

Comment:
To keep backward compatibility, please leave the method. It can be call `BiDiUtils#localeIsRTL`